solidity/contracts/utility/ContractRegistryClient.sol

```diff
@@ -11,6 +11,7 @@ contract ContractRegistryClient is Owned, Utils {
     bytes32 internal constant BANCOR_NETWORK = "BancorNetwork";
     bytes32 internal constant BANCOR_FORMULA = "BancorFormula";
     bytes32 internal constant CONVERTER_FACTORY = "ConverterFactory";
+    bytes32 internal constant CONVERSION_PATH_FINDER = "ConversionPathFinder";

```

Comment:
@yudilevi : So shouldn't we replace `BancorNetworkPathFinder` with `ConversionPathFinder` conclusively?

Reply:
BTW, it is definitely on the "Network-Level", so I'm not sure we should go with "ConversionPathFinder" to begin with.

Reply:
already opened a task for it -
https://trello.com/c/RLI1EQrC/328-ethereum-rename-bancor-network-path-finder-to-conversionpathfinder